docs/modules/ROOT/partials/install/bootstrap-nodes.adoc

```diff
@@ -146,15 +201,27 @@ done && echo -e "\nCilium config updated"
 kubectl -n cilium rollout restart ds/cilium
 ----
 
-. Review and merge the LB hieradata MR (listed in Terraform output `hieradata_mr`) and run Puppet on the LBs after the deploy job has completed
+ifeval::["{provider}" == "cloudscale"]
+. Add Infra Node IPs to LB Hieradata
 +
 [source,bash]
 ----
-for fqdn in "${LB_FQDNS[@]}"; do
-  ssh "${fqdn}" sudo puppetctl run
-done
-----
+git clone git@git.vshn.net:appuio/appuio_hieradata.git
```

Comment:
The terraform checkout of this repo uses the project bot user via https,  and trying to do anything in it asks for the bot's password, which is annoying to script for.
I may be missing something. But I found it easier to just re-clone with ssh...

Reply:
You could maybe do something like

```
git remote add origin-git git@git.vshn.net:appuio/appuio-hieradata.git
```

in the Terraform checkout and then

```
git push origin-git master
```

but that's not easier than just cloning the repo again.

Reply:
yeah... plus, in my local directory the terraform-managed checkout wasn't in a clean state. Whether that was a result of me doing things wrong or a normal possibility, I'm not sure. But a fresh clone certainly never has issues with unclean git trees...